docs/apm/correlations.asciidoc

```diff
@@ -94,7 +99,7 @@ closer to a diagnosis.
 
 [discrete]
 [[correlations-customize-fields]]
-====  Customize fields
+=====  Customize fields
```

Comment:
In 7.14, customize fields is only relevant to error correlations - transaction latency correlations don't need any fields to be selected as we loop over all relevant fields in the transaction.

Reply:
Thanks!  I'd tried to imply that with the nesting, but I've made it more explicit now with a "tip" and by mentioning the "Error rate" tab in that section's intro